### PR TITLE
First draft of landing page and MAUI intro

### PR DESCRIPTION
**Pull request by bijington**
First attempt at creating the shared landing page plus a MAUI Community Toolkit introduction

### CODE REVIEW

File: communitytoolkit/TOC.yml [hunk 1]
```diff
@@ -1,2 +1,8 @@
...
-  href: index.md
\ No newline at end of file
+- name: Community Toolkits Documentation
+  href: index.md
+- name: .NET
+  href: https://docs.microsoft.com/en-gb/windows/communitytoolkit/
+- name: Windows
+  href: https://docs.microsoft.com/en-gb/windows/communitytoolkit/
+- name: MAUI
+  href: maui/index.md
```
Comment (brminnick) [inline communitytoolkit/TOC.yml:8]: Let's add `Community Toolkit` to ensure that users don't think they'll be taken to the .NET Docs / Windows Docs / MAUI Docs.

Let's also remove the culture (in this case en-gb) from Microsoft links. The Microsoft website will automatically add the culture, localizing the website based on the viewer's locale.

```suggestion
- name: .NET Community Toolkit
  href: https://docs.microsoft.com/windows/communitytoolkit/
- name: Windows Community Toolkit
  href: https://docs.microsoft.com/windows/communitytoolkit/
- name: MAUI Community Toolkit
  href: maui/index.md
```

**Review by brminnick — CHANGES_REQUESTED** [review]
Thanks Shaun! These are off to a great start!!

**Comment by bijington** [comment]
@brminnick some issues that were raised against these docs is that we don't include an `ms.prod` or `ms.service` tag in the header. These docs aren't for a specific Microsoft product or service so I assume we want to get this rule turned off?

image

File: communitytoolkit/maui/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The MAUI Community Toolkit is a collection of reusable elements for application development with [.NET MAUI][maui-url], including animations, behaviors, converters, effects, and helpers. It simplifies and demonstrates common developer tasks when building iOS, Android, macOS and WinUI applications using [.NET MAUI][maui-url].
+
+The MAUI Community Toolkit is available as a set of NuGet Packages for new or existing [.NET MAUI][maui-url] projects.
+
+You can also preview the capabilities of the toolkit by running the sample app available in the MAUI Community Toolkit repo.
+
+Feel free to browse the documentation using the table of contents on the left side of this page.
+
+[maui-url]: https://docs.microsoft.com/en-us/dotnet/maui/ "Microsoft MAUI documentation"
```
[the remark this answers is not in the document]
Comment (davidbritch) [reply communitytoolkit/maui/index.md:19]: Also, always remove FQDNs to docs.microsoft.com e.g. https://docs.microsoft.com/dotnet/maui/ becomes /dotnet/maui.

Using FQDNs to docs.microsoft.com breaks something called the air-gapped cloud.

**Comment by davidbritch** [comment]
> @brminnick some issues that were raised against these docs is that we don't include an `ms.prod` or `ms.service` tag in the header. These docs aren't for a specific Microsoft product or service so I assume we want to get this rule turned off?

Nope. We need to insert the correct ms.prod and ms.technology values. They enable us to easily filter docs telemetry. Leave that with me.

**Comment by bijington** [comment]
> > @brminnick some issues that were raised against these docs is that we don't include an `ms.prod` or `ms.service` tag in the header. These docs aren't for a specific Microsoft product or service so I assume we want to get this rule turned off?
> 
> Nope. We need to insert the correct ms.prod and ms.technology values. They enable us to easily filter docs telemetry. Leave that with me.

@davidbritch thank you. I assume you aware but just to confirm the initial `index.md` page applies to multiple products/technologies:
- .NET
- Windows -> UWP
- MAUI

File: communitytoolkit/maui/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bijington) [reply communitytoolkit/maui/index.md]: @davidbritch thanks for that explanation, I have also removed FQDNs from the `TOC.yml` files, I assume this was safe?

File: communitytoolkit/index.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+
+The Community Toolkits offer a collection of reusable implementations that aim to simplify common development tasks for a variety of .NET frameworks.
+
+Learn how to use these .NET Toolkits, view sample code and more.
+
+- ## [.NET Community Toolkit](https://docs.microsoft.com/en-gb/windows/communitytoolkit/)
+
+- ## [Windows Community Toolkit](https://docs.microsoft.com/en-gb/windows/communitytoolkit/)
+
+- ## [MAUI Community Toolkit](maui/index.md)
```
[the remark this answers is not in the document]
Comment (bijington) [reply communitytoolkit/index.md]: I removed the bullets instead here and added a little one liner to hopefully place emphasis on what each toolkit provides.

File: communitytoolkit/maui/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidbritch) [reply communitytoolkit/maui/index.md]: Yep.

**Comment by davidbritch** [comment]
> > @brminnick some issues that were raised against these docs is that we don't include an `ms.prod` or `ms.service` tag in the header. These docs aren't for a specific Microsoft product or service so I assume we want to get this rule turned off?
> 
> Nope. We need to insert the correct ms.prod and ms.technology values. They enable us to easily filter docs telemetry. Leave that with me.

Update: I know what the correct ms.prod value is, but I also need to get an ms.technology value created. It'll be next week before that comes through.

**Comment by bijington** [comment]
> > > @brminnick some issues that were raised against these docs is that we don't include an `ms.prod` or `ms.service` tag in the header. These docs aren't for a specific Microsoft product or service so I assume we want to get this rule turned off?
> 
> > 
> 
> > Nope. We need to insert the correct ms.prod and ms.technology values. They enable us to easily filter docs telemetry. Leave that with me.
> 
> 
> 
> Update: I know what the correct ms.prod value is, but I also need to get an ms.technology value created. It'll be next week before that comes through.

That's great thanks David! Have a great weekend

**Comment by michael-hawker** [comment]
I think we want to use the yml format for the index to get the nice boxes for each project for the main starting point links, like the C# page, eh? https://github.com/dotnet/docs/blob/main/docs/csharp/index.yml

image

Unless we also want to experiment and see what happens if we try the hub page type: https://github.com/dotnet/docs/blob/main/docs/index.yml#L11

In either case, think it'd be good to have our mission statement at the top in the intro:

> The home for .NET based Community Toolkits which are partnering the community with engineers, Microsoft MVPs, and the teams working on those technologies.

We should also call out that they're part of the .NET Foundation.

**Comment by davidbritch** [comment]
YML format is definitely better for the index, but it should be a landing page rather than a hub page. Hub pages are for collections of related products/services. So .NET is the hub page. .NET Community Toolkit should be landing page (like https://github.com/dotnet/docs-maui/blob/main/docs/index.yml).

**Comment by bijington** [comment]
@michael-hawker thanks for the comments. Sorry I meant to include you in the PR review but didn't sort that yet. I'm happy to go with the YAML page but unless we can convince @davidbritch and probably others to get the hub style page I will go with a landing page. It's a shame as the hub style would look great with our setup.

And I'll make sure to include the dotnet foundation part.

**Comment by bijington** [comment]
> YML format is definitely better for the index, but it should be a landing page rather than a hub page. Hub pages are for collections of related products/services. So .NET is the hub page. .NET Community Toolkit should be landing page (like https://github.com/dotnet/docs-maui/blob/main/docs/index.yml).
> 
> 

Thank you for the clarification and example of what to follow

**Comment by bijington** [comment]
Additional note for myself:

Add a note to the top of the docs that let's the user know that the docs are a work-in-progress and also that we are migrating the Windows Toolkit docs

**Comment by bijington** [comment]
> Hub pages are for collections of related products/services. So .NET is the hub page.

We have 3 toolkits under one repo. Are we labelling them as one or could they technically be 3 related products/services? 😄 

I guess this comes in to this `ms.prod` requirement right?

**Comment by bijington** [comment]
@davidbritch is there any documentation on what can go in to one of these YAML files?

**Comment by bijington** [comment]
@michael-hawker and @davidbritch this landing page approach actually looks great! Thanks for the links/suggestion David/Michael it is a big improvement in terms of layout!

landing-page

P.S. I'll get the top item on the left to match the title of the page :)

**Comment by davidbritch** [comment]
> > Hub pages are for collections of related products/services. So .NET is the hub page.
> 
> We have 3 toolkits under one repo. Are we labelling them as one or could they technically be 3 related products/services? 😄
> 
> I guess this comes in to this `ms.prod` requirement right?

It comes down to the URL they are published under. If you published under docs.microsoft.com/CommunityToolkit/ you can have a hub page there. If you publish under docs.microsoft.com/dotnet/... you can't have a hub page as /dotnet/ already has the hub page. Anything under /dotnet/ has to be a landing page.

**Comment by davidbritch** [comment]
Also, it should always be .NET MAUI. Never just MAUI. Then, on first use on every page it should always be .NET Multi-platform App UI (.NET MAUI).

**Comment by bijington** [comment]
> > > Hub pages are for collections of related products/services. So .NET is the hub page.
> > 
> > 
> > We have 3 toolkits under one repo. Are we labelling them as one or could they technically be 3 related products/services? 😄
> > I guess this comes in to this `ms.prod` requirement right?
> 
> It comes down to the URL they are published under. If you published under docs.microsoft.com/CommunityToolkit/ you can have a hub page there. If you publish under docs.microsoft.com/dotnet/... you can't have a hub page as /dotnet/ already has the hub page. Anything under /dotnet/ has to be a landing page.

Thanks for the explanation, it all helps to understand the logic. Actually the landing page looks pretty good with the column based format so I think we can achieve the look we want with that.

**Comment by bijington** [comment]
> Also, it should always be .NET MAUI. Never just MAUI. Then, on first use on every page it should always be .NET Multi-platform App UI (.NET MAUI).

OK great thanks. This looks like something we should add to the contributing guide.

**Comment by bijington** [comment]
Closing due to the inability to continue editing this branch. See #5 for the correct PR

**Pull request closed without merging**